90zfsbootmenu/zfs-chroot.sh

```diff
@@ -17,19 +17,17 @@ if mountpoint="$( allow_rw=yes mount_zfs "${selected}" )"; then
   mount -t proc proc "${mountpoint}/proc"
   mount -t sysfs sys "${mountpoint}/sys"
   mount -B /dev "${mountpoint}/dev"
-  mount -B /tmp "${mountpoint}/var/tmp"
+  mount -B /tmp "${mountpoint}/tmp"
```

Comment:
I never noticed that this was already bound on `/var/tmp`, but I think `/tmp` is more intuitive.